.github/workflows/release.yml

```diff
@@ -78,3 +77,41 @@ jobs:
       uses: softprops/action-gh-release@v2
       with:
         files: ./dist/*
+
+  build-docs:
+    name: Build the Docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+      - name: Install dependencies
+        run: |
+          pip install sphinx sphinx_rtd_theme
```

Comment:
Would it make sense to pin the version here? Seems low risk to not, but also would be an annoying headache to someone if sphinx has a breaking change.

Reply:
good point. moved to requirements.txt